src/guide/advanced/metrics.md

```diff
@@ -12,7 +12,7 @@ configuration.
 
 To use metrics, you need to configure the `/metrics` endpoint in the
 [Iroha configuration](https://github.com/hyperledger/iroha/blob/iroha2-dev/docs/source/references/config.md).
-By default, the endpoint is exposed at `127.0.0.1:8180/metrics`. If the
+By default, the endpoint is exposed at [`127.0.0.1:8180/metrics`](http://127.0.0.1:8180/metrics). If the
```

Comment:
I don't think that makes sense though. These are not meant to be actual clickable links.

Reply:
Wouldn't those be active on a default run, though?

Reply:
I'd say that since there are times when this is an inactive link, it shouldn't be a link at all. Generally in docs these are are not clickable (though I've seen examples of the opposite as well).
But I also see your point, and if you think this is a better user experience, then sure.

Reply:
Just writing it with protocol should work, e.g.

```md
http://127.0.0.1:8180/metrics

127.0.0.1:8180/metrics
```

http://127.0.0.1:8180/metrics

127.0.0.1:8180/metrics

As you see, GitHub parses it as an anchor. VitePress does the same.

---

I think highlight is good because it is more accessible to the user (e.g. it is focusable and easily copyable).